blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sourcegraph 3.31 release"
+publishDate: 2021-08-20T10:00-07:00
+description: "Sourcegraph 3.31 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.31"
+published: false
+heroImage: 
```

Comment:
@iskyOS @rebeccadee - here is the social image/hero image https://www.figma.com/file/BRedkGL1VSHDOjM2ci4L0Q/SG_Evergreen_Blog_Template?node-id=0%3A1